Source/driver/_cpp_parameters

```diff
@@ -525,6 +525,45 @@ reset_checkpoint_step        int           -1
 
 
 
+
+#-----------------------------------------------------------------------------
+# category: particles
+#-----------------------------------------------------------------------------
+
+# permits tracer particle calculation to be turned on and off
+do_tracer_particles          int           0
+
+# whether the local densities at given positions of particles are stored in output files
+timestamp_density            int           0
```

Comment:
it looks like the old default value of this was `1` -- we should keep the defaults unchanged